contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -9,6 +9,17 @@ Feature gates are intended to cover the development life cycle of a feature - th
 Features generally progress through `Alpha` -> `Beta` -> `GA`. Sometimes we end up deciding that a feature is not going to be supported and we end up marking them as `Deprecated`.
 
 The majority of features will go through all three stages, but occasionally there are features which may skip stages.
+While some exceptions may happen, approvers should use the following guidance:
+- features that involve API changes must progress through all `Alpha`, `Beta`, `GA` stages
+- larger or more complex changes should progress through all `Alpha`, `Beta`, `GA` stages
+- smaller changes that carry non-trivial risk (e.g. due to changing user-facing behavior) might skip
+  `Alpha` and start directly in `Beta` (provided the appropriate `Beta` quality is achieved)
+  but should be off by-default until proven in some production environment
+- smaller changes with low enough risk might skip `Alpha` and start directly in `Beta`
```

Comment:
those are basically the bug fixes? Is the KEP only needed to introduce the Feature Gate instead of providing config options?

Reply:
If so, we can be open about it

Reply:
Yes, these are "bugfixes which might need a backout"

Reply:
Maybe just say it like this:

```suggestion
- smaller changes with low enough risk that still may need to be disabled might skip `Alpha` and start directly in `Beta`
```

Reply:
just want it to be very explicit